docs/install.rst

```diff
@@ -0,0 +1,31 @@
+.. _topics-install:
+
+=======
+Install
+=======
+
+.. note:: As a general rule, it's recommended to use an isolation tool such as ``virtualenv``.
```

Comment:
Good point, I think we should add the suggestion as a step and let them choose what tool they want to use since there are alternatives to virtualenv